libs/asset-viewer/src/lib/state/asset-search/asset-search.selector.ts

```diff
@@ -60,12 +60,16 @@ export const selectAssetSearchTotalResults = createSelector(assetSearchFeature,
 export const selectCurrentAssetDetail = createSelector(assetSearchFeature, (state) => state.currentAsset);
 export const selectStudies = createSelector(assetSearchFeature, (state) => state.studies);
 
-export const selectHasDefaultFilters = createSelector(
+export const selectHasDefaultFilters = createSelector(assetSearchFeature, ({ query }) => hasDefaultFilters(query));
+
+export const selectIsDefaultFilterState = createSelector(
   assetSearchFeature,
-  ({ query, currentAsset }) =>
-    currentAsset == null && Object.values(query).every((value) => value === undefined || value == false)
+  ({ query, currentAsset }) => currentAsset == null && hasDefaultFilters(query)
 );
 
+const hasDefaultFilters = (query: AssetSearchQuery): boolean =>
```

Comment:
I am still not fully happy with the naming of these. 
I would prefer something like "isQueryEmpty" or "hasNoActiveFilters". 
And for `selectIsDefaultFilterState` i would suggest something like `isInitialSearchState`. I feel like this more accurately represents the status of the state

Reply:
Agreed. I changed the selectors to: 
- `selectHasDefaultFilters` -> `selectHasNoActiveFilters`
- `selectIsDefaultFilterState` -> `selectIsSearchQueryEmpty`

For the second one, I didn't want to use something like `isInitialSearchState`, as that may implicate that the search has not been executed yet. `selectIsSearchQueryEmpty` makes sense to me here in that neither the search nor a specific assets are set.